api/envoy/extensions/transport_sockets/http_11_proxy/v3/upstream_http_11_connect.proto

```diff
@@ -16,30 +16,28 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 // [#protodoc-title: Upstream HTTP/1.1 Proxy]
 // [#extension: envoy.transport_sockets.http_11_proxy]
 
-// The HTTP/1.1 proxy transport socket opens an upstream connection to a specified proxy address
-// rather than the target host's address. If this transport socket is configured and proxy
-// information is configured, then:
+// HTTP/1.1 proxy transport socket establishes an upstream connection to a proxy address
+// instead of the target host's address. This behavior is triggered when the transport
+// socket is configured and proxy information is provided.
 //
-// * Upstream connections to the proxy address will have a raw HTTP/1.1 CONNECT header prefaced to
-//   the payload, and 200 response stripped (if less than 200 bytes).
-// * Plaintext HTTP/1.1 connections will be sent with a fully qualified URL.
+// Behavior when proxying:
+// =======================
+// * Upstream connections to the proxy include a raw HTTP/1.1 CONNECT header before
+//   the payload. A 200 response is stripped if it's under 200 bytes.
+// * Plaintext HTTP/1.1 connections are sent with a fully qualified URL.
```

Comment:
What does this mean?  Connections don't have URLs, so I'm not sure what this is trying to say.

Reply:
I mentioned what happened in a different comment.